app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -33,18 +49,51 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreateOptionsMenu(menu: Menu): Boolean {
         super.onCreateOptionsMenu(menu)
-        val item = menu.add(getString(R.string.action_logout))
-        item.setIcon(android.R.drawable.ic_menu_search)
-        item.setShowAsAction(MenuItem.SHOW_AS_ACTION_NEVER)
+        menuInflater.inflate(R.menu.logged_in, menu)
         return true
     }
 
+    override fun onPrepareOptionsMenu(menu: Menu?): Boolean {
```

Comment:
The logout button being here seems temporary anyway, until a full settings screen is built, so all of this will have to be pulled out of here eventually.